.gitlab-ci.yml

```diff
@@ -8,7 +8,7 @@ variables:
   ECR_REGISTRY: "${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com"
 
 default:
-  image: "${ECR_REGISTRY}/idp/ci:latest"
+  image: "${ECR_REGISTRY}/idp/ci@sha256:8650f43d0e6d44729bff92657b59e511c874103ae6207b3c0781df042df6932f"
```

Comment:
should we do this via env var as well so we don't need a source code change every time? esp since the registry is already an env var?

Reply:
Sure. Done.